src/fava/core/attributes.py

```diff
@@ -71,6 +74,11 @@ def load_file(self) -> None:
         currency_ranker = ExponentialDecayRanker()
         payee_ranker = ExponentialDecayRanker()
 
+        self.account_details = {
```

Comment:
We don't need this copy here, we can just pass the ledger.accounts dictionary to the frontend.